packages/web/src/pages/saved-page/SavedPageProvider.tsx

```diff
@@ -449,7 +475,9 @@ class SavedPage extends PureComponent<SavedPageProps, SavedPageState> {
       onFilterChange: this.onFilterChange,
       onSortChange: this.onSortChange,
       formatMetadata: this.formatMetadata,
-      getFilteredData: this.getFormattedData,
+      getFilteredData: this.state.allTracksFetched
+        ? this.getFilteredData
+        : this.getFormattedData,
```

Comment:
Not sure if done, but can we have a comment for this function, just like when its used and what it does?